### PR TITLE
Made PingPoll more clear

### PR DESCRIPTION
**Pull request by JustIanJ**
# Summary
Made PingPoll more clear, because it said to choose a time, but we did not know for what

## Checklist

- [ ] If code changes were made then they have been tested
- [x] This PR improves a feature
- [ ] This PR adds a feature
  - [ ] This PR adds a slash command
  - [ ] This PR adds a view
  - [ ] This PR adds a cog
- [x] This PR fixes an issue
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by codeman1o1** [comment]
Please add a summary.

**Comment by JustIanJ** [comment]
It was not clear what i should have done the first time i used it

**Comment by JustIanJ** [comment]
> Please add a summary.

i did just now

**Comment by codeman1o1** [comment]
> > Please add a summary.
> 
> i did just now

In the first message please

File: views/dropdowns/pingpoll_time.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def __init__(self):
         ]
 
         super().__init__(
-            placeholder="Please select a time",
+            placeholder="Please select how long the PingPoll should stay.",
```
Comment (codeman1o1) [inline views/dropdowns/pingpoll_time.py:15]: Please remove the dot at the end of the sentence
```suggestion
            placeholder="Please select how long the PingPoll should stay",
```

**Review by codeman1o1 — CHANGES_REQUESTED** [review]

**Review by codeman1o1 — APPROVED** [review]
LGTM

**Comment by codeman1o1** [comment]
I don't know whether _choose_ or _select_ would be better here.
What is your opinion @JustIanJ?

**Comment by JustIanJ** [comment]
I chose the current one in my branch with a reason

**Comment by codeman1o1** [comment]
> with a reason

What is the reason if I may ask?

**Comment by JustIanJ** [comment]
Because select suggest (to me) that there are a (fairly) limited amount of possible options.
Choose is like a ton of options

**Comment by codeman1o1** [comment]
Makes sense yeah

**Pull request merged**